sdk/tests/conformance2/renderbuffers/framebuffer-test.html

```diff
@@ -191,6 +198,127 @@
+                                       gl.FRAMEBUFFER_ATTACHMENT_COLOR_ENCODING);
+  wtu.glErrorShouldBe(gl, gl.INVALID_OPERATION,
+            "calling getFramebufferAttachmentParameter with pname = GL_FRAMEBUFFER_ATTACHMENT_COLOR_ENCODING on read framebuffer without any attachment should generate INVALID_OPERATION.");
+  gl.getFramebufferAttachmentParameter(gl.DRAW_FRAMEBUFFER,
+                                       gl.COLOR_ATTACHMENT0,
+                                       gl.FRAMEBUFFER_ATTACHMENT_COLOR_ENCODING);
+  wtu.glErrorShouldBe(gl, gl.NO_ERROR,
+            "calling getFramebufferAttachmentParameter with pname = GL_FRAMEBUFFER_ATTACHMENT_COLOR_ENCODING on draw framebuffer should succeed.");
+  gl.framebufferTexture2D(gl.DRAW_FRAMEBUFFER, gl.COLOR_ATTACHMENT0, gl.TEXTURE_2D, null, 0);
+  gl.getFramebufferAttachmentParameter(gl.DRAW_FRAMEBUFFER,
+                                       gl.COLOR_ATTACHMENT0,
+                                       gl.FRAMEBUFFER_ATTACHMENT_COLOR_ENCODING);
+  wtu.glErrorShouldBe(gl, gl.INVALID_OPERATION,
+            "calling getFramebufferAttachmentParameter with pname = GL_FRAMEBUFFER_ATTACHMENT_COLOR_ENCODING on draw framebuffer without any attachment should generate INVALID_OPERATION.");
+
+  // attach/detach a renderbuffer to one framebuffer binding point, while no attachment to the other binding point.
+  function attachAndDetachRenderbuffer(targetA, targetB) {
+    gl.framebufferRenderbuffer(targetA, gl.COLOR_ATTACHMENT0, gl.RENDERBUFFER, rb);
+    wtu.glErrorShouldBe(gl, gl.NO_ERROR,
+            "attaching a renderbuffer to a read/draw framebuffer should succeed.");
+    gl.getFramebufferAttachmentParameter(targetA,
+                                       gl.COLOR_ATTACHMENT0,
+                                       gl.FRAMEBUFFER_ATTACHMENT_COLOR_ENCODING);
+    wtu.glErrorShouldBe(gl, gl.NO_ERROR,
+            "calling getFramebufferAttachmentParameter with pname = GL_FRAMEBUFFER_ATTACHMENT_COLOR_ENCODING on read/draw framebuffer should succeed.");
+    gl.getFramebufferAttachmentParameter(targetB,
+                                       gl.COLOR_ATTACHMENT0,
+                                       gl.FRAMEBUFFER_ATTACHMENT_COLOR_ENCODING);
+    wtu.glErrorShouldBe(gl, gl.INVALID_OPERATION,
+            "calling getFramebufferAttachmentParameter with pname = GL_FRAMEBUFFER_ATTACHMENT_COLOR_ENCODING on read/draw framebuffer without any attachment should generate INVALID_OPERATION.");
+    gl.framebufferRenderbuffer(targetA, gl.COLOR_ATTACHMENT0, gl.RENDERBUFFER, null);
+    wtu.glErrorShouldBe(gl, gl.NO_ERROR,
+            "detaching a renderbuffer from a read/draw framebuffer should succeed.");
+  }
+
+  var readDrawFB1 = gl.createFramebuffer();
+  gl.bindFramebuffer(gl.FRAMEBUFFER, readDrawFB1);
+  attachAndDetachRenderbuffer(gl.READ_FRAMEBUFFER, gl.DRAW_FRAMEBUFFER);
+  attachAndDetachRenderbuffer(gl.DRAW_FRAMEBUFFER, gl.READ_FRAMEBUFFER);
```

Comment:
Same basic problem here, where the same framebuffer object is bound to both the READ and DRAW attachment points.